crypto/util/jwk.go

```diff
@@ -30,13 +30,14 @@ func MapToJwk(jwkAsMap map[string]interface{}) (jwk.Key, error) {
 	if err != nil {
 		return nil, err
 	}
-	return set.Keys[0], nil
+	key, _ := set.Get(0)
+	return key, nil
 }
 
 // MapsToJwkSet transforms JWKs in map structures to a JWK set, just like MapToJwk.
-func MapsToJwkSet(maps []map[string]interface{}) (*jwk.Set, error) {
-	set := &jwk.Set{Keys: make([]jwk.Key, len(maps))}
-	for i, m := range maps {
+func MapsToJwkSet(maps []map[string]interface{}) (jwk.Set, error) {
```

Comment:
I thought I saw a `MapToJWK` and back again somewhere in another file. Duplicate code?

Reply:
https://github.com/nuts-foundation/nuts-node/pull/95